src/js/control-bar/time-controls/time-display.js

```diff
@@ -63,7 +63,12 @@ class TimeDisplay extends Component {
    * @private
    */
   updateTextNode_() {
-    if (this.textNode_) {
+    if (!this.contentEl_) {
+      return;
+    }
+
+    // ie8/9 don't support `el.contains` inside of setTimeout call.
+    if (this.textNode_ && this.textNode_.parentNode === this.contentEl_) {
```

Comment:
what about just removing everything inside of `contentEl_`? It'll also fix an issue where if the page is translated in chrome using google translate, the text node will be wrapped in another element and this method will fail.
This could also just happen in a separate PR.